ROS_API.md

```diff
@@ -26,4 +26,8 @@ Use `bringup.launch.py` from `rosbot_bringup` to start all base functionalities
   - `/tf_static` (_tf2_msgs/TFMessage_)
   - `/_motors_cmd` (_std_msgs/Float32MultiArray_)
   - `/rosbot_base_controller/odom` (_nav_msgs/Odometry_)
-  - `/imu_broadcaster/imu` (_sensor_msgs/Imu_)
\ No newline at end of file
+  - `/imu_broadcaster/imu` (_sensor_msgs/Imu_)
+
+Use `simulation.launch.py` from `rosbot_gazebo` to start all base functionalities for ROSbot 2, 2 PRO, 2R in the Gazebo simulator.
+> **Warning**
+> The distance sensors' topics types from Gazebo simulation mismatch with the real ones. The real type is [sensor_msgs/msg/Range](https://github.com/ros2/common_interfaces/blob/rolling/sensor_msgs/msg/Range.msg) but simulated [sensor_msgs/msg/LaserScan](https://github.com/ros2/common_interfaces/blob/rolling/sensor_msgs/msg/LaserScan.msg). The first value of the `ranges` in [sensor_msgs/msg/LaserScan](https://github.com/ros2/common_interfaces/blob/rolling/sensor_msgs/msg/LaserScan.msg) is the `range` field of [sensor_msgs/msg/Range](https://github.com/ros2/common_interfaces/blob/rolling/sensor_msgs/msg/Range.msg).
```

Comment:
I think that it will be also worth mentioning why there is a mismatch (that Gazebo doesn't support Range sensors yet).